### PR TITLE
Router: Explicitly build types

### PR DESCRIPTION
**Pull request by Tobbe**
Makes type generation run when you manually build just the router package. See this comment for more context https://github.com/redwoodjs/redwood/pull/1641#discussion_r569810369

### CODE REVIEW

**Review by dac09 — APPROVED** [review]

**Comment by thedavidprice** [comment]
Do we need any other approval on this? If no, please go ahead and merge.

**Comment by Tobbe** [comment]
@thedavidprice I think it's fine to merge this. It was just I who missed adding the ttsc command when I rewrote everything in TS. All the other TS packages has it, so this one should too. So just an oversight on my part.

If you still want to wait for Peter's approval that's 100% fine, but if you want to go ahead and merge this I think that should be fine too.

**Comment by thedavidprice** [comment]
Roger that and background makes sense! When publishing, we handle TS build separately so it was already being included. I’ll get this one in. Thanks!

**Pull request merged**
